### ISSUE TITLE
Overviewer not finding textures with 1.13.2.jar ?

### ISSUE THREAD

**Issue by ghost** [mide/minecraft-overviewer#29]
The 1.13.2.jar downloads but then overviewer doesn't find it

```
$ docker run   --rm   -e MINECRAFT_VERSION="1.13.2"   -v /Users/[me]/Desktop/[minecraft_server]:/home/minecraft/server/:ro   -v /Users/[me]/Desktop/[minecraft_server_map]:/home/minecraft/render/:rw   mide/minecraft-overviewer:latest
Using Client URL https://launcher.mojang.com/v1/objects/30bfe37a8db404db11c7edf02cb5165817afb4d9/client.jar.
WARNING: timestamping does nothing in combination with -O. See the manual
for details.

--2019-02-28 01:28:01--  https://launcher.mojang.com/v1/objects/30bfe37a8db404db11c7edf02cb5165817afb4d9/client.jar
Resolving launcher.mojang.com (launcher.mojang.com)... 54.192.147.10
Connecting to launcher.mojang.com (launcher.mojang.com)|54.192.147.10|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 16089640 (15M) [application/java-archive]
Saving to: '1.13.2.jar'

     0K .......... .......... .......... .......... ..........  0% 2.79M 5s
[spliced]
 15700K .......... ..                                         100% 8.04M=4.6s

2019-02-28 01:28:05 (3.31 MB/s) - '1.13.2.jar' saved [16089640/16089640]

2019-02-28 01:28:06  Welcome to Minecraft Overviewer!
2019-02-28 01:28:15  Generating textures...
*******************************************************************************
2019-02-28 01:28:15 E Could not find the textures while searching for 'assets/minecraft/textures/block/grass_block_top.png'. Try specifying the 'texturepath' option in your config file.
Set it to the path to a Minecraft Resource pack.
Alternately, install the Minecraft client (which includes textures)
Also see <http://docs.overviewer.org/en/latest/running/#installing-the-textures>
(Remember, this version of Overviewer requires a 1.13-compatible resource pack)
(Also note that I won't automatically use snapshots; you'll have to use the texturepath option to use a snapshot jar)
```

**Comment by athimel** [comment]
Same here. I've tried a lot of solutions, but cannot make it work :(

(BTW It has been working for months before 1.13.x, many thanks)

**Comment by patricklucas** [comment]
Adding `texturepath = "/home/minecraft/VERSION.jar"` to `config.py` seems to make it work, and it should be pretty straightforward to do that automatically in the entrypoint script.

**Comment by mide** [comment]
Solved by #31.
